### DISCUSSION TITLE
蓝灯免费版升级专业版付款后未生效是咋回事，也没有发激活码

### DISCUSSION THREAD

**Discussion by hnzyxiaozhe** [getlantern/lantern-forum-cn#683] [Q&A]
蓝灯免费版升级专业版付款后未生效是咋回事，也没有发激活码
![QQ图片20240703205421](https://github.com/getlantern/lantern-forum-cn/assets/72728930/9d15b99e-45e2-4a49-b8a7-1d58c572fc43)

**Comment by MaxMKMax** [comment]
@hnzyxiaozhe 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by hnzyxiaozhe** [reply]
什么时候能办理好，已经提交相关资料 @Mike777Khoo

**Comment by hnzyxiaozhe** [comment]
已在连接提交邮箱和支付单据~




肖哲
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月4日(星期四) 凌晨0:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 蓝灯免费版升级专业版付款后未生效是咋回事，也没有发激活码 (Discussion #683)





 
@hnzyxiaozhe 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
 后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
 
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by hnzyxiaozhe** [reply]
![QQ图片20240706104049](https://github.com/getlantern/lantern-forum-cn/assets/72728930/27da8cd6-d22f-47af-bc93-44a97988ec64)
发送的激活码仍然无法激活
